buildSrc/src/main/groovy/com.virtuslab.gitmachete.buildsrc/IntellijVersionHelper.groovy

```diff
@@ -0,0 +1,26 @@
+package com.virtuslab.gitmachete.buildsrc
+
+class IntellijVersionHelper {
+    static String getFromBuildNumber(String buildNumber) {
+        return '20' + buildNumber.substring(0, 2) + '.' + buildNumber.substring(2, 3)
+    }
+
+    static String toBuildNumber(String version) {
+        return version.substring(2, 6).replace('.', '')
+    }
+
+    static String getMajorPart(String version) {
+        return version.substring(0, 6)
+    }
+
+    static Properties getProperties() {
+        Properties properties = new Properties()
+        properties.load(getFile().newDataInputStream())
+        return properties
+    }
+
+    static File getFile() {
```

Comment:
`private`? so as to enforce that all interactions with the file are via this class